builtin/providers/aws/data_source_aws_acm_certificate.go

```diff
@@ -0,0 +1,46 @@
+package aws
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/aws/aws-sdk-go/service/acm"
+	"github.com/hashicorp/errwrap"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func dataSourceAwsAcmCertificate() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceAwsAcmCertificateRead,
+		Schema: map[string]*schema.Schema{
+			"domain": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"arn": &schema.Schema{
```

Comment:
Should we also have an optional `statuses` attribute here, so that users can filter for only certificates that are in a good state?

```hcl
data "aws_acm_certificate" "example" {
  domain   = "example.com"
  statuses = ["issued"]
}
```